pipeline/ingest/behavior_ingest.py

```diff
@@ -0,0 +1,89 @@
+import datajoint as dj
+
+from pipeline import lab, experiment
+from pipeline import get_schema_name
+from . import session_loaders
+
+schema = dj.schema(get_schema_name('behavior_ingest'))
+
+# ============== SETUP the LOADER ==================
+
+try:
+    data_dir = dj.config['custom']['data_root_dir']
+except KeyError:
+    raise KeyError('Unspecified data root directory! Please specify "data_root_dir" under dj.config["custom"]')
+
+try:
+    session_loader_class = dj.config['custom']['session_loader_class']
+except KeyError:
+    raise KeyError('Unspecified session loader method! Please specify "session_loader_method" under dj.config["custom"]')
+
+if session_loader_class in session_loaders.__dict__:
```

Comment:
rather than `__dict__` use `dir()` to list content, and then use `getattr(module, name)` to fetch batch the name.

Reply:
done